frontend/src/components/ConfirmatioStep/ConfirmationStep.js

```diff
@@ -6,27 +6,36 @@ const messages = defineMessages({
   confirmationMessage: {
     id: 'app.customFeedback.confirmation.message',
     description: 'Thank you for your feedback! The window will close shortly.'
+  },
+  skippedMessage: {
+    id: 'app.customFeedback.windowWillClose',
+    description: 'The window will close shortly.'
   }
 });
 
-const ConfirmationStep = ({ intl, getRedirectUrl, getRedirectTimeout }) => {
+const ConfirmationStep = ({ intl, getRedirectUrl, getRedirectTimeout, endReason, isSkipped }) => {
   useEffect(() => {
-    const redirectTimeout = getRedirectTimeout ? getRedirectTimeout() : null;
+    const redirectTimeout = getRedirectTimeout ? getRedirectTimeout() : 10000;
     const timer = setTimeout(() => {
       const redirectUrl = getRedirectUrl ? getRedirectUrl() : null;
       if (redirectUrl) {
         window.location.href = redirectUrl;
       } else {
-        window.close();
+        if (!isSkipped) {
+            window.close();
+        }
```

Comment:
The window should still close when skipped
```suggestion
        window.close();
```